lib/matplotlib/tests/test_ticker.py

```diff
@@ -1449,6 +1449,18 @@ def test_latex(self, is_latex, usetex, expected):
             assert fmt.format_pct(50, 100) == expected
 
 
+@pytest.mark.xfail(locale != "fy_DE.UTF-8",
+                   reason="Locale 'fy_DE.UTF-8' is not supported")
+def test_locale_comma():
+    currentLocale = locale.getlocale()
+    locale.setlocale(locale.LC_ALL, 'fy_DE.UTF-8')
+    ticks = mticker.ScalarFormatter(useMathText=True, useLocale=True)
+    fmt = '$\\mathdefault{%1.1f}$'
+    x = ticks._format_maybe_minus_and_locale(fmt, 0.5)
+    locale.setlocale(locale.LC_ALL, currentLocale)
+    assert x == '$\\mathdefault{0{,}5}$'
```

Comment:
```suggestion
    fmt = '$\\mathdefault{%1.1f}$'
    x = ticks._format_maybe_minus_and_locale(fmt, 0.5)
    assert x == '$\\mathdefault{0{,}5}$'
    # Do not change , in the format string
    fmt = ',$\\mathdefault{,%1.1f},$'
    x = ticks._format_maybe_minus_and_locale(fmt, 0.5)
    assert x == ',$\\mathdefault{,0{,}5},$'
    locale.setlocale(locale.LC_ALL, currentLocale)
```
To test that a `,` in the format string is not changed as well.

(Not sure if the locale must be reset, but if so, do that last.)